### DISCUSSION TITLE
Full access

### DISCUSSION THREAD

**Discussion by Subteras** [sandboxie-plus/Sandboxie#1461] [Q&A Feedback]
I would like to know if it is possible to give the sandbox full access to files, registry, etc. I just can't run Rockstar game launcher  in the sandbox.

**Comment by DavidXanatos** [comment]
yes its possible but then its no longer sandboxed so what would be the point?
OpenPipePath=*
OpenConfPath=*

**Reply by Subteras** [reply]
I only need it to run two versions of applications on a computer. At the same time, isolation is not so important, the programs are tested and safe.

**Reply by Subteras** [reply]
Tell please these commands need to be entered in .ini file?

**Reply by isaak654** [reply]
> Tell please these commands need to be entered in .ini file?

Yes, exactly.
OpenConfPath command is available since Plus v1.0.0 / Classic 5.55.0 onwards.
